.gitignore

```diff
@@ -329,3 +329,6 @@ ASALocalRun/
 
 # MFractors (Xamarin productivity tool) working folder
 .mfractor/
+
+# Ignore generated style file
+src/View.Uno/Generated/mergedpages.xaml
```

Comment:
The file is called "mergedpages.xaml", but it's not merging any **page**.
You could use "MergedResourceDictionaries.xaml".